### ISSUE TITLE
[MRESOLVER-104] maven-resolver-demo-maven-plugin uses reserved artifactId

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-resolver#858]
**[Elliotte Rusty Harold](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=elharo)** opened **[MRESOLVER-104](https://issues.apache.org/jira/browse/MRESOLVER-104?redirect=false)** and commented

```
[INFO] --- maven-plugin-plugin:3.5.2:descriptor (default-descriptor) @ maven-resolver-demo-maven-plugin ---
[ERROR]

Artifact Ids of the format maven-___-plugin are reserved for 
plugins in the Group Id org.apache.maven.plugins
Please change your artifactId to the format ___-maven-plugin
In the future this error will break the build.
```

---

**Remote Links:**
- [GitHub Pull Request #73
](https://github.com/apache/maven-resolver/pull/73)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

The problem is likely here that `maven` apears twice in the name!

Here is the code for: https://github.com/apache/maven-plugin-tools/blob/26ba0323bfd66214222b1be4d71402b0948d51bf/maven-plugin-plugin/src/main/java/org/apache/maven/plugin/plugin/AbstractGeneratorMojo.java#L199-L207

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [28e4ec02dc94690828c0abcaaf8686c9e080f182](https://gitbox.apache.org/repos/asf?p=maven-resolver.git;a=commit;h=28e4ec02dc94690828c0abcaaf8686c9e080f182).

workaround = instead of maven-resolver-demo artifactId, switched to resolver-demo
